mypy/checkexpr.py

```diff
@@ -1375,8 +1376,67 @@ def visit_type_application(self, tapp: TypeApplication) -> Type:
         return AnyType()
 
     def visit_type_alias_expr(self, alias: TypeAliasExpr) -> Type:
+        """ Get type of a type alias (could be generic) in a runtime expression."""
+        item = alias.type
+        if (isinstance(item, (Instance, TupleType, UnionType, CallableType))
+           and not alias.runtime):
+            item = self.replace_tvars_any(item)
+        if isinstance(item, Instance):
+            tp = type_object_type(item.type, self.named_type)
+        else:
+            # TODO: Better error reporting: need to find line for
+            # unsubscribed generic aliases, that are invalid at runtime.
```

Comment:
@gvanrossum  Yes, before the error was pointing to the line where type alias was defined, not to line where it was illegally used. Now  I fixed this, so that the error points to exact place and says what is wrong. This could show a funny error: ``"Type alias to Callable" not callable``, but this is actually true. If one tries ``typing.Callable()`` one will get exception at runtime.